playground/components/header.tsx

```diff
@@ -118,8 +129,11 @@ export const Header: ParentComponent<{
           }}
           title="Github"
         >
-          <Show when={context.dark()} fallback={<Icon class="h-6" viewBox="0 0 96 96" path={githubMark} />}>
-            <Icon viewBox="0 0 96 96" class="h-6" path={githubMarkWhite} />
+          <Show
```

Comment:
added separate fill for dark and light mode

Reply:
I think you can simplify the entire `Show` component to just: 
`<Icon viewBox="0 0 96 96" class="h-6" path={githubMark} />`

Also, I wonder whether it makes sense to also add the class `cursor-alias` to the `a` tag above. It might make it a bit clearer that this will open in a new tab

Reply:
@modderme123 
Removed the `Show` component, added `cursor-alias` class to `a` tag and also made the SVG path object inline.